Python/PerformanceTest_Agility.py

```diff
@@ -234,13 +234,13 @@ def COMwk(totF_Z, totF_Y, totF_X, mass, landings):
     velo_x = []
    # for ii, val in enumerate(landings[:(len(landings)-1)]):
     for ii, val in enumerate(zip(landings, takeoffs)):
-        veloz = integrate.cumtrapz(acc_z[val[0]:val[1]], dx = 1/frequency)
+        veloz = integrate.cumulative_trapezoid(acc_z[val[0]:val[1]], dx = 1/frequency)
```

Comment:
was .cumtrapz just archived?.. not sure if we want to start switching libraries which I think cumulative_trapezoid is calling on torch. cumtrapz should work well with numpy arrays which is a majority of what we work with.